src/articles/_posts/2016-07-20-pwa-desktop.md

```diff
@@ -0,0 +1,72 @@
+---
+title: 'Progressive web apps running as native OS X apps'
+authors:
+- vlad-filippov
+intro: 'Progressive Web Apps are getting ready for desktop.'
+tags:
+- pwa
+license: cc-by-3.0
+published: false
+---
+
+<figure block="figure">
+	<img elem="media" src="{{ page.id }}/native-pwa.jpg" alt="Progressive Web Apps running as separate native apps on OS X">
```

Comment:
@mathiasbynens  Wanna change this to: `<img elem="media" src="{{ page.id }}/multi-platform.jpg" alt="Progressive Web Apps running as separate native apps">` and remove`native-pwa.jpg` from PR per feedback?